src/main/java/com/hw14/patterns/fabricmethod/FabricMethodPattern.java

```diff
@@ -0,0 +1,21 @@
+package com.hw14.patterns.fabricmethod;
+
+import com.hw14.patterns.fabricmethod.Builder.Lesson;
+import com.hw14.patterns.fabricmethod.Proxy.Education;
+import com.hw14.patterns.fabricmethod.Proxy.ProxySelfEducation;
+import com.hw14.patterns.fabricmethod.Singleton.Singleton;
+
+public class FabricMethodPattern {
+    public Course create(String typeOfLesson) {
```

Comment:
The same comment as for class AbstractFactory. Please fix

Reply:
+